zone/attack.cpp

```diff
@@ -3827,28 +3827,28 @@ void Mob::CommonDamage(Mob* attacker, int64 &damage, const uint16 spell_id, cons
 		const auto has_npc_given_event = (
 			(
 				IsNPC() &&
-				parse->HasQuestSub(CastToNPC()->GetNPCTypeID(), EVENT_DAMAGE_GIVEN)
+				parse->HasQuestSub(CastToNPC()->GetNPCTypeID(), EVENT_DAMAGE_GIVEN, true)
```

Comment:
Seems like we should check encounters regardless, this is a very strange smell that maybe we should cut early

Reply:
I'd remove the optional bool and just make it default behavior

Reply:
Sounds good.